### PR TITLE
Orchestrated 10L Int5 record stack, LeakyReLU² toggle, RunPod helpers

### PR DESCRIPTION
**Pull request by rithvik-duddupudi**
## Summary
Adds a competitive **10L int5/int6** trainer snapshot under `records/track_10min_16mb/2026-03-21_OrchestratedStack_10LInt5/` (orchestrated stack aligned with the 10L Int5 MLP line), plus supporting scripts and tests.

## Trainer / record
- Mixed int5 (MLP) / int6 (attn), SmearGate, BigramHash, Muon+WD, SWA; sliding eval defaults (`EVAL_BATCH_SEQS=64`).
- **`LEAKY_RELU_SLOPE`** env for ReLU² vs LeakyReLU² MLP ablation.
- **`SMOKE_MODE`**: skip val during train + skip export/final eval for cheap infrastructure checks (not leaderboard metrics).
- Default `DATA_PATH` / `TOKENIZER_PATH` resolved relative to **repo root** so `torchrun` works from any cwd; clearer errors if tokenizer/dataset missing.

## Repo ergonomics
- `requirements-orchestrated-stack.txt` at repo root (main deps + `zstandard`).
- `scripts/smoke_orchestrated_8xh100.sh`, `scripts/run_orchestrated_full_ab.sh`, `scripts/run_orchestrated_stack_8xh100.sh`.
- CPU tests: `tests/test_orchestrated_mlp_activation.py`, `tests/test_orchestrated_smoke_hyperparams.py`.
- Planning docs: `docs/PLAN-h100-novel-budget.md`, `docs/PLAN-leaderboard-novel-improve.md`; workflow doc updates.

## Verify
```bash
.venv/bin/python -m unittest tests.test_orchestrated_mlp_activation tests.test_orchestrated_smoke_hyperparams -v
```

### CODE REVIEW

**Comment by MatoTeziTanka** [comment]
### Community Review — Orchestrated 10L Int5 record stack, LeakyReLU² toggle, RunPod helpers

**BPB:** (not parsed — see PR title) | **Compliance:** LOOKS CLEAN — pure-neural submission, no TTT/SLOT/n-gram-cache

**What I found in the code** (head SHA `d4da6d7e0e26`, file `records/track_10min_16mb/2026-03-21_OrchestratedStack_10LInt5/train_gpt.py`):

Static code review found no TTT adaptation function, no SLOT optimization loop, no n-gram-cache class, and no pre-quant val-token fine-tune. The eval path uses the standard sliding-window stride-64 pattern. The submission is a pure-neural architecture iteration on the standard SP1024/SP4096/SP8192 baseline.

**CPU smoke test (CT2038 proteus-engine, 2026-04-11):** import OK in 0.06s, dim=512, layers=10, vocab=1024, code=56255 B, SMOKE_TEST_PASS

**Verdict:** LOOKS CLEAN.

**Recommendation to @cocohearts @valerio-oai @0hq @yuzhougu-oai @notapplica:** **MERGE** pending the usual record-track checks (3-seed validation, under-16MB artifact cap, ≤600s train + ≤600s eval on 8×H100 SXM). No compliance flags from the classification pass — this looks like a clean pure-neural iteration on the standard baseline.

*Auto-classification caveat:* this review was drafted by the AST-based classifier. If there's a non-standard eval mechanism (logit postprocessing, hedge mixing, etc.) that I missed because it's factored into a helper file or a non-standard function name, please flag it and I'll re-run the audit manually.

---
*Reviewed by [@MatoTeziTanka](https://github.com/MatoTeziTanka) — [The Agora](https://matotezitanka.github.io/parameter-golf). CPU smoke test (CT2038 proteus-engine, 2026-04-11): import OK in 0.06s, dim=512, layers=10, vocab=1024, code=56255 B, SMOKE_TEST_PASS. Classification via deterministic AST-based `classify_prs.py` (pattern bank derived from ~65 manually-reviewed PRs earlier in the 2026-04-11 sweep). This review was auto-drafted from a template and spot-checked before posting — if the template misread your code, please call it out so I can iterate the classifier.*
